sw/ground_segment/python/gvf/gvfFormation.py

```diff
@@ -7,9 +7,10 @@
 import numpy as np
 import sys
 from os import path, getenv
+PPRZ_HOME = getenv("PAPARAZZI_HOME", path.normpath(path.join(path.dirname(path.abspath(__file__)), '../../../../')))
 PPRZ_SRC = getenv("PAPARAZZI_SRC", path.normpath(path.join(path.dirname(path.abspath(__file__)), '../../../../')))
 sys.path.append(PPRZ_SRC + "/sw/lib/python")
-sys.path.append(PPRZ_SRC + "/sw/ext/pprzlink/lib/v1.0/python")
+sys.path.append(PPRZ_HOME + "/var/lib/python")
```

Comment:
oki. There will be soon a new update. I will include your suggestion by then :P.